server/http/src/service.rs

```diff
@@ -346,7 +352,7 @@ where
                 Ok(response) => Ok(response),
                 Err(err @ GraphQLServerError::ClientError(_)) => Ok(Response::builder()
                     .status(400)
-                    .header(CONTENT_TYPE, "text/plain")
+                    .header(CONTENT_TYPE, "text/html")
```

Comment:
`text/plain` here was correct, I think. The `err` just contains random text, it's not marked up as HTML